### ISSUE TITLE
The Economy & Trade table in the sector view is inconsistent with the Commodity Market

### ISSUE THREAD

**Issue by lolo101** [pioneerspacesim/pioneer#5882]
### Observed behaviour

The Economy & Trade table in the sector view is inconsistent with the Commodity Market in the single station of the system
![Capture d’écran du 2024-08-03 10-00-30](https://github.com/user-attachments/assets/08d2c8c7-b4b3-4820-9f03-d9cc26f98df8)
![Capture d’écran du 2024-08-03 09-59-48](https://github.com/user-attachments/assets/dbed1c15-be78-4e8c-8e42-e3db09ae093c)

### Expected behaviour

The Commodity Market import/export icons are consistent with the system's Economy & Trade table

Note that the Economy & Trade showed the same icons when I was in Gliese 117 since I relied on them to buy a large quantity of Liquid O2 to sell them in Arshir. But the price in Arshir is lower than the place where I bought it

### Steps to reproduce

Compare Economy & Trade table in the sector view and the Commodity Market in the current station

__My pioneer version (and OS):__

20230314 (Linux)

__My output.txt (required) and game save (optional, but recommended)__

[_autosave3.gz](https://github.com/user-attachments/files/16478957/_autosave3.gz)

**Comment by impaktor** [comment]
Isn't this just the usual confusion of system vs. station level import/export (not a bug)? #5836

**Comment by lolo101** [comment]
It looks like the same issue indeed. However, the Ashir system has a single station so I don't understand why station & system levels mismatch in that case.

Furthermore, the price of Liquid O2 is lower than the price in the system I bought it (Gliese 117) whereas the trade was shown as profitable and is actually not

**Comment by Web-eWorks** [comment]
The system economy and the economy of a station within that system are two different things regardless of how many stations exist within a system. This is because the sector map economy view only operates on the level of the system-wide economy, and the individual economies for stations within that system are not generated within the context it is able to inspect.

From an in-universe perspective, the difference between economy levels is because there are *always* more economic actors in a system than the single spaceport you're allowed to land at and that is visible to the player in the context of the game.

Station-level economies always *modify* the system-wide economic factors. You can switch to the system map view and inspect the import/export level of a specific station within the system to confirm your trades. In this case, I suspect if you were to look at the system-wide economy of Gliese 117, you'd find that the system as a whole is majorly dependent on importing Liquid Oxygen, or if you looked at the station you sold the Liquid Oxygen to you'd see that it has a very strong local production of Liquid Oxygen and exports to the surrounding area (and thus isn't strongly interested in buying the commodity).

I'd definitely like to add some more functionality to the trade computer in the future. For example, being able to take data about nearby systems (whether gathered from exploration or purchased from an agency) and find stations to sell goods at, or to highlight stations in the system outline which would allow you to sell specific items in your cargo hold for more than you paid for them. However, I can't provide any estimate on when work might be done in this area.

**Comment by lolo101** [comment]
Ok understood. Maybe this is not an issue and can be closed 🙂
However, I believe the current visual design makes it confusing for new players : icons are identical in system view and in station and there is no explicit indication of the level of imports/exports.

**Comment by bszlrd** [comment]
Maybe reword the text in the sector map? 
"System-wide trade analysis of Sol" for example might clarify it, and should take roughly the same room.
